app/brave_settings_strings.grdp

```diff
@@ -1440,6 +1440,17 @@ Leo. This action cannot be undone.
     Data and chat history
   </message>
 
+  <!-- Settings / Playlist -->
+  <message name="IDS_SETTINGS_PLAYLIST_ENABLE_PLAYLIST_LABEL" desc="The text for setting to en/disable Playlist feature">
+    Enable Playlist
+  </message>
+  <message name="IDS_SETTINGS_PLAYLIST_CACHE_BY_DEFAULT_LABEL" desc="The text for setting to en/disable media cache behavior">
+    Automatically keep files for offline
+  </message>
+  <message name="IDS_SETTINGS_PLAYLIST_CACHE_BY_DEFAULT_SUB_LABEL" desc="A description for setting to en/disable media cache behavior">
+    This option will automatically cache media files for items so you can play them without an internet connection.
```

Comment:
Suggestion: `This option will automatically cache media files so that you can play them without an internet connection.`

Reply:
`This option will automatically cache media files so you can play without an internet connection.`